lochness/__init__.py

```diff
@@ -376,9 +379,9 @@ def listdir(Lochness, d):
 def attempt(f, Lochness, *args, **kwargs):
     '''attempt a function call'''
     if len(attempt.warnings) >= 5:
-        lochness.email.attempts_error(Lochness, attempt)
+        # lochness.email.attempts_error(Lochness, attempt)
         attempt.warnings = []
-        #raise AttemptsError('too many attempt warnings')
+        # raise AttemptsError('too many attempt warnings')
```

Comment:
Is it intentionally or inadvertently commented out?

Reply:
Great point. This line will be uncommented in the new commit.

Reply:
20882d3